sections/dashboard/MobileDashboard/MobileDashboard.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC } from 'react';
 
 import { CompetitionBanner } from 'sections/shared/components/CompetitionBanner';
-import { zeroBN } from 'utils/formatters/number';
+import { zeroBN } from 'sdk/utils/number';
```

Comment:
I noticed you created a ZERO_WEI in constants, would be good to replace zeroBN with that everywhere as it's always been a confusing name representing wei instead of BN. Not sure if you want do that in a follow up or just in this PR.

Reply:
Renamed in latest commit.